examples/async-structured-outputs.py

```diff
@@ -11,7 +12,7 @@ class FriendInfo(BaseModel):
 
 
 class FriendList(BaseModel):
-  friends: list[FriendInfo]
```

Comment:
this generates the following error with python 3.8.

```bash
tomoyafujita@~/DVT/03_OSS/github.com/ollama/ollama-python >python ./examples/async-structured-outputs.py
Traceback (most recent call last):
  File "./examples/async-structured-outputs.py", line 14, in <module>
    class FriendList(BaseModel):
  File "./examples/async-structured-outputs.py", line 15, in FriendList
    friends: list[FriendInfo]
TypeError: 'type' object is not subscriptable
```

introduced by https://github.com/ollama/ollama-python/pull/354, see more details for https://peps.python.org/pep-0585/.